api/v1beta1/helmchart_types.go

```diff
@@ -45,9 +45,20 @@ type HelmChartSpec struct {
 	// +required
 	Interval metav1.Duration `json:"interval"`
 
-	// Alternative values file to use as the default chart values, expected to be a
-	// relative path in the SourceRef. Ignored when omitted.
+	// Alternative list of values files to use as the default chart values,
+	// expected to be a relative path in the SourceRef. Values files are
+	// merged in the order of this list with the last file overriding the
+	// first. The default values.yaml is not included by default. Ignored
```

Comment:
I feel that

> The default values.yaml is not included by default.

is essential information that should come first, before the further description of behavior.

This is especially important to people "scan reading", as they will often skip once the rest of a part of text when they think they know how it behaves.

Reply:
Agreed. Re: scanning, in any examples we should include `values.yaml` with a `values-production.yaml`. I think that would help infer that you need both.

I initially figured the `values.yaml` would be included always until I went and read the source code (which most won't do).

Reply:
Feel free to directly reword my documentation. I'm doing my best to be clear and concise but I won't take it personally if you wish to reword all of it.